### DISCUSSION TITLE
Project Steering Committee Elections 2021

### DISCUSSION THREAD

**Discussion by swainn** [tethysplatform/tethys#701] [Announcements]
All:

It's time again for Tethys Platform Steering Committee member elections! We didn't have elections last year due to COVID, so two of our committee positions are up for re-election: Term 1 and Term 2. For more information about the terms, see our project by-laws document: [Tethys Platform Project By-Laws](http://www.tethysplatform.org/project-steering-committee)

I'd like to thank Scott Christensen and Sarva Pulla for their service in these positions over the last few years.

We'd like to open nominations to refill these positions for the next term cycle. Please reply to this post with the names (preferably @mention) of those that you would like to nominate for the steering committee. Nominees must indicate that they accept the nomination to be considered (e.g. with a post saying "I accept" or with a "thumbs up" reaction on the post).

Nominations will be open until June 28, 2021 at midnight UTC

--
Tethys Platform 
Project Steering Committee
Nathan Swain
Scott Christensen
Sarva Pulla

**Comment by swainn** [comment]
I nominate @sdc50

**Comment by msouff** [comment]
I nominate @rileyhales

**Comment by swainn** [comment]
I nominate @shawncrawley

**Comment by swainn** [comment]
I nominate @ezrajrice
